pkg/security/module/server.go

```diff
@@ -447,6 +475,12 @@ func (a *APIServer) SendStats() error {
 			}
 		}
 	}
+
+	if count := atomic.SwapInt64(&a.expiredProcessEvents, 0); count > 0 {
```

Comment:
Will this field be always properly aligned? Elsewhere in the agent there is a push to use `uber-go/atomic` instead of `sync/atomic` to avoid alignment issues altogether.

Reply:
Thanks for pointing this out @vickenty . I've prepared a separate PR to fix this:
https://github.com/DataDog/datadog-agent/pull/12203